README.markdown

```diff
@@ -132,7 +132,7 @@ Where `X.Y.Z` is replaced with the desired version. For example:
 - run: jsonschema fmt path/to/schemas --check
 ```
 
-### From NPM
+### From npm
```

Comment:
This is how it is spelled allover the https://docs.npmjs.com. Also here: https://docs.npmjs.com/about-npm

Reply:
Makes sense!